packages/create-atomic/package.json

```diff
@@ -40,12 +40,11 @@
     "@coveo/platform-client": "60.4.0",
     "abortcontroller-polyfill": "1.7.5",
     "detect-indent": "7.0.1",
-    "https-proxy-agent": "5.0.1",
-    "inquirer": "9.3.7",
+    "https-proxy-agent": "7.0.6",
     "isomorphic-fetch": "3.0.0",
     "minimist": "1.2.8",
     "node-plop": "^0.32.0",
-    "plop": "3.1.2"
+    "plop": "4.0.1"
```

Comment:
drops support for old node versions

https://github.com/plopjs/plop/releases/tag/plop%404.0.0